js/src/cleantalk-comments-editscreen.js

```diff
@@ -11,6 +11,7 @@ jQuery(document).ready(function(){
 		jQuery('.column-author a, .comment-author a').each(function(){
 			var ct_curr_str = jQuery(this).html();
 			if(ct_is_email(ct_curr_str) != -1 || ct_is_ip(ct_curr_str) != -1){
+				ct_curr_str += '?utm_source=admin_side&utm_medium=comments&utm_content=avatar&utm_campaign=apbct_links';
```

Comment:
Just want to say - hard coded links in JS is the bad way.

Reply:
Yeah, let's change it with next task.